scipy/stats/stats.py

```diff
@@ -6964,12 +6964,14 @@ def kstest(rvs, cdf, args=(), N=20, alternative='two-sided', mode='auto'):
     N : int, optional
         Sample size if `rvs` is string or callable.  Default is 20.
     alternative : {'two-sided', 'less', 'greater'}, optional
-        Defines the alternative hypothesis.
+        Defines the null and alternative hypotheses.
         The following options are available (default is 'two-sided'):
 
           * 'two-sided'
-          * 'less': one-sided, see explanation in Notes
-          * 'greater': one-sided, see explanation in Notes
+          * 'less'
```

Comment:
maybe we can even drop the bullet points?

```
alternative : {'two-sided', 'less', 'greater'}, optional
   Defines the null and alternative hypotheses. Please see explanations in the Notes below.
```

Reply:
one still needs to mention the default

Reply:
Sure.